airbyte-integrations/connectors/source-facebook-marketing/acceptance-test-config.yml

```diff
@@ -21,6 +21,6 @@ tests:
       future_state_path: "integration_tests/future_state.json"
   full_refresh:
     - config_path: "secrets/config.json"
-      # TODO Change below `configured_catalog_without_insights.json` to `configured_catalog.json` after October 7 2021
-      #  because all running campaigns should be finished by that time.
-      configured_catalog_path: "integration_tests/configured_catalog_without_insights.json"
+      configured_catalog_path: "integration_tests/configured_catalog.json"
+      ignored_fields:
+        "ads_insights_age_and_gender": [ "cost_per_estimated_ad_recallers", ]
```

Comment:
This config is specific to your case, please make it more general, like described in this issue https://github.com/airbytehq/airbyte/issues/6433